### PR TITLE
{vis}[intel/2015a] Graphviz 2.38.0 + pygraphviz 1.3rc2 w/ Python 2.7.10 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**
nothing too special other that version updates

requires ~~#1861~~

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review?

**Comment by boegel** [comment]
Jenkins: test this please

File: easybuild/easyconfigs/g/GObject-Introspection/GObject-Introspection-1.44.0-intel-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+homepage = 'https://wiki.gnome.org/GObjectIntrospection/'
+description = """GObject introspection is a middleware layer between C libraries
+ (using GObject) and language bindings. The C library can be scanned at
+ compile time and generate a metadata file, in addition to the actual
+ native C library. Then at runtime, language bindings can read this
+ metadata and automatically provide bindings to call into the C library."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+source_urls = ['http://ftp.gnome.org/pub/gnome/sources/%(namelower)s/%(version_major_minor)s/']
```
Comment (JensTimmerman) [inline easybuild/easyconfigs/g/GObject-Introspection/GObject-Introspection-1.44.0-intel-2015a.eb]: isn't there a constant for this?

File: easybuild/easyconfigs/p/Pango/Pango-1.37.2-intel-2015a.eb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+homepage = 'http://www.pango.org/'
+description = """Pango is a library for laying out and rendering of text, with an emphasis
+ on internationalization. Pango can be used anywhere that text layout
+ is needed, though most of the work on Pango so far has been done in
+ the context of the GTK+ widget toolkit. Pango forms the core of text
+ and font handling for GTK+-2.x."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+source_urls = ['http://ftp.gnome.org/pub/GNOME/sources/pango/%(version_major_minor)s/']
```
Comment (JensTimmerman) [inline easybuild/easyconfigs/p/Pango/Pango-1.37.2-intel-2015a.eb]: isn't there a constant for this? (use %(namelower)s if not?)

Comment (boegel) [inline easybuild/easyconfigs/p/Pango/Pango-1.37.2-intel-2015a.eb]: it's already there, nice catch

```
$ eb --avail-easyconfig-templates | grep GNOME
    FTPGNOME_SOURCE: http download for gnome ftp server (http://ftp.gnome.org/pub/GNOME/sources/%(namelower)s/%(version_major_minor)s)
```

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 7 out of 10 (10 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/be6e9ece52bccba162b7 for a full test report.

**Comment by boegel** [comment]
Failed because of:

```
Module 'fontconfig/2.11.1-intel-2015a' conflicts with the currently loaded module(s) 'fontconfig/2.11.94-intel-2015a'
```

but the unit tests are supposed to capture this, yet they didn't... Crap.

**Comment by boegel** [comment]
Looks like the issue of missing the conflict is caused by #1853

**Comment by boegel** [comment]
#1870 should fix that bug

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 21 out of 21 (15 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/85a0bdcae08de68229b8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 15 out of 15 (15 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/7b98e5fd3c064a843e48 for a full test report.

**Comment by boegel** [comment]
Going in, thanks for the review @JensTimmerman!

**Pull request merged**